torch_xla/experimental/stablehlo_saved_model.py

```diff
@@ -0,0 +1,110 @@
+import copy
+import shutil
+import os
+import re
+import numpy as np
+import torch
+from torch import nn
+import torch_xla
+from torch_xla.core import xla_model as xm
+import tensorflow as tf
+import torchvision
+import torch._dynamo as torchdynamo
+
+from tensorflow.compiler.tf2xla.python import xla as tfxla  # type: ignore[import]
```

Comment:
Why is `# type: ignore[import]` needed here?

Reply:
removed